snap/files/bin/bridge-setup

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -euo pipefail
+log(){ echo "bridge-setup: $*" >&2; }
+die(){ echo "bridge-setup ERROR: $*" >&2; exit 1; }
+
+apply_from_snap_config() {
+  local iface bridge phys
+  iface="$(snapctl get network.interface || true)"
+  bridge="$(snapctl get network.bridge || true)"; : "${bridge:=br-ex}"
+  phys="$(snapctl get network.physnet || true)"; : "${phys:=physnet1}"
+  [[ -n "$iface" ]] || { log "no network.interface set; skipping"; exit 0; }
+  ensure_bridge_and_mapping "${bridge}" "${iface}" "${phys}"
+}
+
+ensure_bridge_and_mapping() {
+  local br="$1" ifc="$2" phys="$3"
+  local ovsctl=""
+  if command -v ovs-vsctl >/dev/null 2>&1; then
+    ovsctl="ovs-vsctl"
+  elif [[ -x /snap/microovn/current/usr/bin/ovs-vsctl ]]; then
```

Comment:
I think the last step would be to actually consume both:

ovn-chassis plug from microovn
and get a bin slot exposing the ovs-vsctl command to ensure we get the right binary (right version)